examples/neural_quantum_states/.gitignore

```diff
@@ -0,0 +1,4 @@
+logger/
```

Comment:
I'm not sure why uploading this file ? This is a file that tells git what to version and what to not version.

Shouldn't we decide exactly what is being versioned on this repo and only push said files ?

Some files named here are actually part of the PR (run.sh,  exp_configs ...), I think we may want to remove some files from this PR.

Reply:
The .gitignore has been removed.